BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java

```diff
@@ -39,6 +45,19 @@ public Drive findById(final @PathVariable Long id) {
     }
 
 
+    @GetMapping("/passengerId/{passengerId}")
+    private List<Drive> findAllByPassengerId(final @PathVariable Long passengerId) {
+        List<PassengerDrive> passengerDriveList = passengerDriveRepository.findAllByPassengerId(passengerId);
+
+        List<Drive> result = passengerDriveList.stream()
```

Comment:
We assume we already have passenger info on the page, correct?

Reply:
Yes